buddyboss/buddyboss-platform
# PROD-771 Activity page doesn't protected by MemberPress- Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2755  .

### How to test the changes in this Pull Request:

1. Go to MemberPress > Memberships
2. Click on "Add New" and create a new membership
3. Go to MemberPress > Rules
4. Click on "Add New" and create a rule
5. Set Activity page in the Protected Content section and assign membership to this rule
6. See activity page is protected now.

### Proof Screenshots or Video

https://www.loom.com/share/990660449d9c4517b0d2657b3ab1d4d2

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
